client/my-sites/plugins/minimum-jet-pack-version-notice.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import notices from 'notices';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	getSite,
+	isJetpackSite,
+	siteHasMinimumJetpackVersion
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends React.Component {
+
+	static propTypes = {
+		translate: React.PropTypes.func.isRequired,
+		minimumJetpackVersionFailed: React.PropTypes.bool.isRequired,
+		site: React.PropTypes.object,
+	};
+
+	notificationHandler( { minimumJetpackVersionFailed, site, translate } ) {
+		if ( minimumJetpackVersionFailed ) {
+			notices.warning(
```

Comment:
I tried to use the new approach to notices and in this case its problematic and using will require some work and creative way to solve the problem. And I think would be something to another pull request.
I will try to summarize the problem with using the new notices, maybe other people are able to see an easy solution.
UserA is in a plugins page of siteA that has minimum Jetpack version, than switches to siteB that has no minimum Jetpack version. The action to change the site is triggered than our notices component detects the currentSelectedSite id changed and immediately trigger a notice saying siteB has no minimum Jetpack version. Meanwhile a route change action is dispatched (after the site change action) and the new notices by default are cleared once the route changes so the notice just appears for half a second.
The new notices component has a flag that allows notices to persist to the next route but if we use that we are creating another problem. If the user goes directly to siteB and then switches to siteA the notice will continue.
If you find have a simple solution in mind please tell me if not I can try to think of something in another PR, I think it requires the addiction of some logic to solve this problem .